.env.test

```diff
@@ -1 +1 @@
-ALLOWED_DOMAINS=some_random_domain,http://localhost
\ No newline at end of file
+ALLOWED_DOMAINS=some_random_domain,.*\.geo\.admin\.ch,http://localhost
```

Comment:
at second though, I don't know if it is a good idea to have "real" domains here, and not test with mocked up values.
it would mean that if we fail to properly configure our service down the k8s line, it could still function somewhat properly and we wouldn't detect our lack of config.
What do you think?

Reply:
Those values are not part of the code but environment variable that we set only for unit testing. I would agree not to use read domain as default environment variable but I don't see any issue for testing. Moreover the `geo.admin.ch` is not an internal address that we should not disclosed like some other addresses.